Sources/StreamVideoSwiftUI/Utils.swift

```diff
@@ -9,6 +9,7 @@ public class Utils {
     public var userListProvider: UserListProvider
     public var callSoundsPlayer = CallSoundsPlayer()
     internal var videoRendererFactory = VideoRendererFactory()
+    internal let pictureInPictureAdapter = StreamPictureInPictureAdapter()
```

Comment:
I'm thinking to actually make this a separate injection key from utils. The reason being that utils, can be created and we may end up with more than one instances of the adapter. WDYT?

Reply:
yes, that's a good idea 👍